neptune/experiments.py

```diff
@@ -61,9 +61,11 @@ class Experiment(object):
         Column sorting
     """
 
-    def __init__(self, client, leaderboard_entry):
+    def __init__(self, client, _id, internal_id, project_full_id):
```

Comment:
Project name can change. Shouldn't we store `uuid` instead?

Reply:
`project_full_id` is used only during api errors handling and project name will rarely change so I don't think it's worth it.

Reply:
So `def project_full_id(self):` property should not be exposed in Experiment.